csharp/ql/src/semmle/code/csharp/dataflow/flowsources/PublicCallableParameter.qll

```diff
@@ -14,7 +17,7 @@ private import semmle.code.csharp.frameworks.WCF
  * }
  * ```
  */
-class PublicCallableParameterFlowSource extends DataFlow::ParameterNode {
+deprecated class PublicCallableParameterFlowSource extends DataFlow::ParameterNode {
```

Comment:
The reason was that this class is not used anywhere, and I doubt any customers use it. I think a class like this would be better in for example a data flow tutorial.

Reply:
OK, let's keep it then.